Cloudinary/CLDConfiguration.swift

```diff
@@ -101,13 +101,17 @@ import Foundation
      */
     public static func initWithEnvParams() -> CLDConfiguration? {
         let dict = ProcessInfo.processInfo.environment
-        if let url = dict[Defines.ENV_VAR_CLOUDINARY_URL] {
+        if let url = dict[Defines.ENV_VAR_CLOUDINARY_URL], validateUrl(url: url) == true {
             return CLDConfiguration(cloudinaryUrl: url)
         }
         
         return nil
     }
     
+    public static func validateUrl(url: String) -> Bool{
```

Comment:
this should be `fileprivate instead of public as it is only used in this file. We try to limit access modifiers  as much as possible because any public Api is locked for the lifetime of the version.

Reply:
Set to `internal` so tests can access